opm/output/data/Wells.hpp

```diff
@@ -356,9 +397,27 @@ namespace Opm {
             auto& comp = this->connections[ i ];
             comp.read(buffer);
         }
+
+        // Segment information (if applicable)
+        const auto nSeg = [&buffer]() -> unsigned int
+        {
+            auto n = 0u;
+            buffer.read(n);
+
+            return n;
+        }();
+
+        for (auto segID = 0*nSeg; segID < nSeg; ++segID) {
+            auto seg = Segment{};
+
+            seg.read(buffer);
+
+            const auto segNumber = seg.segNumber;
```

Comment:
Your vertical spacing seems a bit random - it does not speak to me at least.

Reply:
I'll compress it down